langs/gleam/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM alpine:3.20 AS builder
+
+RUN apk add --no-cache curl
+
+RUN curl -L https://github.com/gleam-lang/gleam/releases/download/v1.2.1/gleam-v1.2.1-x86_64-unknown-linux-musl.tar.gz | tar xz
+
+FROM codegolf/lang-base
+
+COPY --from=0 /bin                     /bin
+COPY --from=0 /cache                   /cache
+COPY --from=0 /lib/ld-musl-x86_64.so.1 /lib/
+
+RUN mkdir -p /gleam/src
+COPY --from=0 gleam.toml /gleam
+COPY --from=0 gleam /usr/bin/
```

These files come directly from the file system, not the builder stage

```suggestion
COPY gleam.toml /gleam
COPY gleam /usr/bin/
```